src/import/FSHImporter.ts

```diff
@@ -970,6 +974,35 @@ export class FSHImporter extends FSHVisitor {
   }
 
   visitAddElementRule(ctx: pc.AddElementRuleContext): AddElementRule {
+    const addElementRule = this.parseNewElement(ctx);
+    addElementRule.types = this.parseTargetType(ctx);
+    addElementRule.types.forEach(onlyRuleType => {
+      if (FLAGS.includes(onlyRuleType.type)) {
+        logger.warn(
+          `The targetType '${onlyRuleType.type}' appears to be a flag value rather than a valid target data type.`,
+          {
+            file: this.currentFile,
+            location: this.extractStartStop(ctx)
+          }
+        );
+      }
+    });
+    return addElementRule;
+  }
+
+  visitAddCRElementRule(ctx: pc.AddCRElementRuleContext): AddElementRule {
+    const addElementRule = this.parseNewElement(ctx);
+    if (ctx.SEQUENCE()) {
+      addElementRule.contentReference = ctx.SEQUENCE().getText();
+    } else if (ctx.CODE()) {
+      addElementRule.contentReference = ctx.CODE().getText();
+    }
```

Comment:
Since `contentReference` is a URL, I think it probably makes sense to use the alias-aware function (I forget its exact name) to extract the value.